github/teams.go

```diff
@@ -476,3 +476,88 @@ func (s *TeamsService) RemoveTeamProject(ctx context.Context, teamID int64, proj
 
 	return s.client.Do(ctx, req, nil)
 }
+
+// GroupObject reprents list of Idp Group.
```

Comment:
Please change this to: `// IDPGroupList represents a list of external identity provider (IDP) groups.`
I know GitHub spells it `IdP`, but Go always uses `ID` for "identification", so let's stick to the Go standard here too.
:smile:

Reply:
should I also change the name of struct to `IDPGroupList` ?

Reply:
Yes, please.